go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/vsphere-csi-driver/v3
 
-go 1.21
+go 1.22.8
```

Comment:
Any particular reason for using the patch version? Can we not use `1.22`

Reply:
Having go.mod set to 1.22 alone gives this error,

```
(vsphere-csi-driver) $ make check
go: downloading go1.22 (darwin/amd64)
go: download go1.22 for darwin/amd64: toolchain not available
Makefile:24: *** Requires Go "go1.13"+ for Go module support.  Stop.
```

reference: https://github.com/golang/go/issues/62278#issuecomment-1698829945

Reply:
Good to know :)